modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -413,9 +425,10 @@ export default class NestedTabs extends Base {
 			elementor.$preview[ 0 ].contentWindow.dispatchEvent( new CustomEvent( 'elementor/elements/link-data-bindings' ) );
 		}
 
-		const targetIndex = event.detail.index + 1 || 1;
```

Comment:
I have made the update of the active tab conditional. I have only done it when there was no active tab.